modules/infinite-scroll/infinity.php

```diff
@@ -1211,8 +1211,6 @@ function filter_infinite_scroll_results( $results, $query_args, $wp_query ) {
 	 * @return string or null
 	 */
 	function query() {
-		global $wp_customize;
```

Comment:
Looks like this isn't a 5.0 change per se, just unused variables still here. Fine to remove here.